components/arm/universalrobots/ur.go

```diff
@@ -453,15 +472,7 @@ func (ua *URArm) MoveToJointPositionRadians(ctx context.Context, radians []float
 			return err
 		}
 
-		if slept > 5000 && !retried {
```

Comment:
This was always very dubious and with the changes Nicolas made to manage the connection it should no longer be necessary.  Ran this by Rand she approves